code/src/cljs/sixsq/nuvla/ui/cimi/views.cljs

```diff
@@ -221,10 +222,9 @@
              $select      :select,
              $aggregation :aggregation,
              $orderby     :orderby} @query-params]
-        [ui/Form {:aria-label   "filter parameters"
-                  :on-key-press (partial forms/on-return-key
-                                         #(when @selected-id
-                                            (dispatch [::events/get-results])))}
+        [ui/Form {:aria-label  "filter parameters"
+                  :on-key-down #(when (= (.-key %) "Enter")
+                                  (.preventDefault %))}
```

Comment:
Searching by pressing Enter after changing a form input is disabled with this change.

I think the right solution here is reverting this change and putting `type="button"` on the FilterButton. That's in `filter_comp/views.cljs`, line 326 add new property:
```:type     :button```